hack/tests/e2e-ansible.sh

```diff
@@ -52,36 +44,35 @@ test_operator() {
     local metrics_test_image="fedora:latest"
 
     header_text "wait for operator pod to run"
-    if ! timeout 1m kubectl rollout status deployment/memcached-operator;
+    if ! timeout 1m kubectl rollout status deployment/osdk-controller-manager;
```

Comment:
```suggestion
   if ! timeout 1m kubectl rollout status deployment/osdk-controller-manager -n ${operator_namespace} ;
```

Reply:
You need pass the operatorNamespace to check that. See that when we build the operator and deploy it then, it will create always the same default ns for the project in the new layout.

Reply:
These (ancient) tests were all hardcoded to the `default` namespace, we could specify namespace but it's not actually configurable without updating more of the test